exercises/bob/.meta/generator/bob_case.rb

```diff
@@ -0,0 +1,9 @@
+require 'generator/exercise_case'
+
+class BobCase < Generator::ExerciseCase
+  def workload
+    indent_lines(["remark = %Q(#{input})",
+      "assert_equal '#{expected}', Bob.hey(remark), %Q(Bob hears #{input}, and..)"
```

Comment:
@Insti using ```%q``` with curly brackets insted of parens did the trick. I had no idea you could use alternative delimiters, thanks!